### PR TITLE
TF-1301 Support select font size while compose email

### PR DESCRIPTION
**Pull request by dab246**
## Issue

#1301 

## Dependent

- Need merged: https://github.com/linagora/html-editor-enhanced/pull/25 

## Resolved

https://github.com/linagora/tmail-flutter/assets/80730648/9fe599a7-6c90-4e65-8759-dc39a67bdd1c

### CODE REVIEW

File: lib/features/composer/presentation/model/rich_text_style_type.dart [hunk 1]
```diff
@@ -19,32 +20,18 @@ enum RichTextStyleType {
...
-      case italic:
-        return 'italic';
-      case underline:
-        return 'underline';
-      case strikeThrough:
-        return 'strikeThrough';
-      case textColor:
-        return 'foreColor';
-      case textBackgroundColor:
-        return 'hiliteColor';
```
Comment (hoangdat) [inline lib/features/composer/presentation/model/rich_text_style_type.dart:35]: why you remove these lines?

File: pubspec.yaml [hunk 2]
```diff
@@ -54,7 +54,7 @@ dependencies:
   html_editor_enhanced:
     git:
       url: https://github.com/linagora/html-editor-enhanced.git
-      ref: email_supported
+      ref: improvement/support-set-font-size-pixel
```
Comment (hoangdat) [inline pubspec.yaml:57]: please create a PR to `email_supported`

File: lib/features/composer/presentation/model/rich_text_style_type.dart (hunk 1 above)
Comment (dab246) [reply lib/features/composer/presentation/model/rich_text_style_type.dart:35]: It is not used

File: pubspec.yaml (hunk 2 above)
Comment (dab246) [reply pubspec.yaml:57]: It's actually already been created

**Review by hoangdat — APPROVED** [review]

**Pull request merged**